tests/Infrastructure.IntegrationTests/Persistence/ApplicationDbContextTests.cs

```diff
@@ -2,19 +2,20 @@
 using System.Linq;
 using System.Threading.Tasks;
 using Codidact.Domain.Entities;
+using Codidact.Infrastructure.IntegrationTests;
 using Codidact.Infrastructure.Persistence;
 using Microsoft.EntityFrameworkCore;
 using Xunit;
 
-namespace Infrastructure.IntegrationTests.Persistence
+namespace Codidact.Infrastructure.IntegrationTests.Persistence
 {
     public class ApplicationDbContextTests
     {
         private readonly ApplicationDbContext _sutContext
             = new ApplicationDbContext(
                 new DbContextOptionsBuilder<ApplicationDbContext>()
                 .UseInMemoryDatabase(Guid.NewGuid().ToString())
-                .Options
+                .Options, new CurrentUserServiceMock()
```

Comment:
This syntax is unclear. I'd suggest putting `new DbContextOptionsBuilder<ApplicationDBContext>().UseInMemoryDatabase(Guid.NewGuid().ToString()).Options` on one line, then `new CurrentUserServiceMock()` on the next with the closing parenthesis.

Reply:
Fixed in https://github.com/codidact/core/pull/29/commits/7eb646f8cbcb25a64a7d085f69f710389e1b3f9c